dags/glam_subdags/extract.py

```diff
@@ -21,13 +23,13 @@ def extracts_subdag(
         SubDagOperator(
             subdag=extract_channel_subdag(
                 dag_id,
-                "extract_{}".format(channel),
+                f"extract_{channel}",
```

Comment:
just curious, is your linter doing the f string swaps?

Reply:
yes

Reply:
E.g. it was also complaining about nested `if`s here, which is why I refactored it
![image](https://github.com/mozilla/telemetry-airflow/assets/1668835/9d063009-f1f2-498c-ac0a-8e211f04f884)